src/app/pages/rpc-send-transfer/rpc-send-transfer-summary.tsx

```diff
@@ -0,0 +1,60 @@
+import toast from 'react-hot-toast';
+import { useLocation } from 'react-router-dom';
+
+import { useClipboard } from '@stacks/ui';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useExplorerLink } from '@app/common/hooks/use-explorer-link';
+
+import { SentSummaryLayout } from '../send/sent-summary/sent-summary.layout';
+import { RpcSendTransferLayout } from './components/rpc-send-transfer.layout';
+
+export function RpcSendTransferSummary() {
+  const { state } = useLocation();
+  const { handleOpenTxLink } = useExplorerLink();
+  const analytics = useAnalytics();
+
+  const {
+    txId,
+    txValue,
+    txFiatValue,
+    txFiatValueSymbol,
+    symbol,
+    txLink,
+    arrivesIn,
+    sendingValue,
+    recipient,
+    fee,
+    totalSpend,
+  } = state;
+
+  const { onCopy } = useClipboard(txId);
+
+  const onClickLink = () => {
+    void analytics.track('view_transaction_confirmation', { symbol: 'BTC' });
+    handleOpenTxLink(txLink);
+  };
```

Comment:
In the component above, function declarations are used, what's the reason to alternate style?